deeplabcut/utils/auxfun_multianimal.py

```diff
@@ -210,36 +186,33 @@ def SaveFullMultiAnimalData(data, metadata, dataname, suffix="_full"):
     return data_path, metadata_path
 
 
-def LoadFullMultiAnimalData(dataname):
-    """Save predicted data as h5 file and metadata as pickle file; created by predict_videos.py"""
-    data_file = dataname.split(".h5")[0] + "_full.pickle"
+def load_multianimal_full_meta_data(
```

Comment:
I have the feeling that we have multiple methods doing the same thing.
For loading the data saved in *_full.pickle and *_meta.pickle, we have this method, but there is also one in _deeplabcut/utils/auxiliaryfunctions.py_ called `load_video_metadata()` (and I added also `load_video_full_data()`)..
Should we unify these, to have them defined only once?